_data/governance.yml

```diff
@@ -0,0 +1,37 @@
+introText: This section is updated every 6 months by the Scala Center.
+cards:
+  - title: "Scala Improvement Process"
+    description: "The SIP is the primary mechanism for evolving the Scala language."
+    expandText: "Process & People"
+    links:
+      - name: SIP Committee Members
+        url: https://docs.scala-lang.org/sips/process-specification.html#the-sip-committee
+      - name: SIP Process Specification
+        url: https://docs.scala-lang.org/sips/process-specification.html
+  - title: "Scala 3 Maintenance"
+    description: "Managing the Scala 3 compiler, standard library and documentation."
+    expandText: "Process & People"
+    links:
+      - name: Core Maintainers
+        url: https://github.com/lampepfl/dotty/blob/main/MAINTENANCE.md#compiler
+      - name: Maintenance Procedures
+        url: https://github.com/lampepfl/dotty/blob/main/MAINTENANCE.md
+  - title: "Tooling"
+    description: "Managing build tools, linters, IDEs, and other tools."
+    expandText: "Process & People"
+    links:
+      - text: "Process and People are TBD, please stay tuned! However you can read below about the Scala Tooling Summit:"
+      - name: "Scala Tooling Summit of March 2023"
+        url: https://www.scala-lang.org/blog/2023/04/11/march-2023-scala-tooling-summit.html
+  - title: "Moderation"
+    description: "Learn how the Scala community is moderated."
```

Comment:
I don’t have a concrete suggestion to make, but I think the description should summarize what we see in the links (the fact that there is a moderation team, a code of conduct, etc.).